client/.babelrc.js

```diff
@@ -51,7 +51,8 @@ const config = {
           'sql',
           'svg',
           'typescript',
-          'xml'
+          'xml',
+          'bash'
```

Comment:
Nitpick - could this go in the right place to keep the list in alphabetical order?